README.md

```diff
@@ -17,7 +17,7 @@ from DAP or Conjur.
 
 ## Supported services
 
-- Openshift 3.9 and 3.11
+- Openshift 3.11 and 3.10
```

Comment:
let's re-add 3.9

Reply:
We aren't yet supporting this with automation so I suggest we don't readd this until v3.9 is fully tested and automation is present
Last week, we left it where @InbalZilberman would look into if we want to still hold this requirement or reduce scope with Naama.